lib/jsrt/JsrtSourceHolder.cpp

```diff
@@ -49,7 +51,6 @@ namespace Js
             *utf8Script = HeapNewArray(utf8char_t, cbUtf8Buffer);
         }
 
-        Assert(length < MAXLONG);
```

Comment:
I'd prefer to keep the Assert. It's not that we every expect the application to crash, but if it did, this Assert serves as documentation of an invariant that was unintentionally broken, and would help more quickly track down the issue.